controllers/openshift_controller.go

```diff
@@ -592,8 +592,8 @@ func (r *KataConfigOpenShiftReconciler) listKataPods() error {
 	}
 	for _, pod := range podList.Items {
 		if pod.Spec.RuntimeClassName != nil {
-			if *pod.Spec.RuntimeClassName == r.kataConfig.Status.RuntimeClass {
-				return fmt.Errorf("Existing pods using Kata Runtime found. Please delete the pods manually for KataConfig deletion to proceed")
+			if contains(r.kataConfig.Status.RuntimeClass, *pod.Spec.RuntimeClassName)  {
```

Comment:
Found by `go fmt ./...` :

```suggestion
			if contains(r.kataConfig.Status.RuntimeClass, *pod.Spec.RuntimeClassName) {
```

Reply:
Dang, not _again_! ;-)

Good catch, fixed, thanks.